locales/en-US.json

```diff
@@ -1,5 +1,6 @@
 {
```

Comment:
Why this file has changes that are unrelated to this feature? ``noMediaSourcesAvailable`` is a duplicate, was moved to the bottom.

Reply:
I used the i18n ally vs-code extension to add them which seems to have sorted them alphabetically. Removed the unrelated changes.